### PR TITLE
Add Issue template

### PR DESCRIPTION
**Pull request by mapedraza**
This PR add issue template to the repo. 

2 templates added:
- Bug report
- Feature request

Additionally, a GH action - [Advanced Issue Labeler](https://github.com/marketplace/actions/advanced-issue-labeler) -  is included, in order to label issues according to the options choose under dropdown selectors in templates.

This issue labeler assigns, for each type of issues:

**Bug report**
NGSI Version. Possible values:
- NGSIv2
- NGSI-LD

**Feature request**
Type. Possible values:
- New feature
- Enhancement
- Techdeb

### CODE REVIEW

**Comment by mapedraza** [comment]
You can check a preview here: 
https://github.com/mapedraza/iotagent-node-lib/issues/new/choose

**Comment by mapedraza** [comment]
Screenshot 2023-06-06 at 10 17 21


Screenshot 2023-06-06 at 10 18 18

**Review by AlvaroVega — APPROVED** [review]
LGTM

File: .github/workflows/issue-labelert.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Issue labeler
```
Comment (fgalan) [inline .github/workflows/issue-labelert.yml:1]: Initial comment explaning the purpose of this file. It's not trivial.

Comment (fgalan) [reply .github/workflows/issue-labelert.yml:1]: Fix typo in file name.

**Comment by fgalan** [comment]
Analize the way of including a "free form" option or a way of cleaning up the markdown before submitting (as currently done in Orion). If possible, only for users with permissions in this repo.

**Comment by mapedraza** [comment]
> Analize the way of including a "free form" option or a way of cleaning up the markdown before submitting (as currently done in Orion). If possible, only for users with permissions in this repo.

Blank issues creation is allowed as you can see in the issue template `config.yml` (general configuration)
https://github.com/telefonicaid/iotagent-node-lib/blob/1f13f581c213cf06d0213426ff4e37bf8d63d50c/.github/ISSUE_TEMPLATE/config.yml#L1-L9

This config option enables a small and quite hidden option as you can see in the following picture:

Screenshot 2023-06-07 at 10 40 32

File: .github/workflows/issue-labelert.yml (hunk 1 above)
Comment (mapedraza) [reply .github/workflows/issue-labelert.yml]: Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1379/commits/e1c9404d075cdc9470ca84ac121df335930d891e

**Comment by mapedraza** [comment]
> The same template will be added to iota-json and iota-ul ?

Yes, once this template is fully reviewed, a subsequent PR to both repos is going to be pushed.

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
Should be applied to IOTA-JSON and IOTA-UL repos.

**Pull request merged**